src/components/search-form.js

```diff
@@ -65,12 +65,11 @@ export default class SearchForm extends Component {
               </div> )
               : null
             }
-            <div>
+            <div className="search__topic">
```

Comment:
does the double underscore mean something?

Reply:
@Br3nda yup this is following BEM methodology.  double underscores is used when you want to specify the element name. `.search` would be the block (B), `__topic` is the element (E) and M is for Modifier. We use `--` for modifiers.

Examples:

```
.search

.search__topic

.search__topic--error
```

Reply:
Thank! i learned something

Reply:
✨

Reply:
Linked to &__topic { '_search.scss`